migration/sql-files/005-add-search-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS fields_title ON work_items USING GIN (to_tsvector('english', fields->>'system.title'));
```

Comment:
@sbose78 Import the larger dataset that is mentioned on the public mailing list. Then it will become very obvious when it doesn't hit the indexes. _Everything_ is fast on a small dataset.